site/docs/configuration.md

```diff
@@ -78,6 +78,14 @@ Iceberg tables support table properties to configure table behavior, like the de
 | --------------------------------------------- | -------- | ------------------------------------------------------------- |
 | compatibility.snapshot-id-inheritance.enabled | false    | Enables committing snapshots without explicit snapshot IDs    |
 
+### Reserved table properties
+Reserved table properties are only used to control behaviors when creating or updating a table.
+The value of these properties are not persisted as a part of the table metadata.
+
+| Property       | Default  | Description                                                   |
+| -------------- | -------- | ------------------------------------------------------------- |
+| format-version | 1        | Table's format version (can be 1 or 2)                        |
+
```

Comment:
How about adding a reference to https://iceberg.apache.org/#spec/#format-versioning ? 😸

Reply:
Initially I thought of adding it and then I ignored thinking it is understandable. I have updated it now.